src/metabase/sso/ldap/default_implementation.clj

```diff
@@ -163,9 +177,9 @@
                          (t2/update! :model/User (:id user) user-changes)
                          (t2/select-one [:model/User :id :last_login :is_active] :id (:id user))) ; Reload updated user
                        user))
-                   (-> (user/create-new-ldap-auth-user! {:first_name first-name
-                                                         :last_name  last-name
-                                                         :email      email})
+                   (-> (create-new-ldap-auth-user! {:first_name first-name
+                                                    :last_name  last-name
+                                                    :email      email})
                        (assoc :is_active true)))]
```

Comment:
could this `assoc` be moved to the `create-new-ldap-auth-user!` function? since its docstring says the new user will be active, feels a bit weird to have that logic be here :)